Makefile

```diff
@@ -152,3 +152,27 @@ runserver: ## Run the Django application locally
 
 .PHONY: dev-init
 dev-init: load-design-system-templates collectstatic makemigrations migrate createsuperuser ## Run the pre-run setup scripts
+
+.PHONY: functional-tests-up
+functional-tests-up:  ## Start the functional tests docker compose dependencies
+	docker compose -f functional_tests/docker-compose.yml up -d
+
+.PHONY: functional-tests-up-dev
+functional-tests-up-dev:  ## Start the functional tests docker compose dependencies and dev app
+	docker compose -f functional_tests/docker-compose-dev-app.yml up -d
+
+.PHONY: functional-tests-down
+functional-tests-down:  ## Stop the functional tests docker compose dependencies
+	docker compose -f functional_tests/docker-compose-dev-app.yml down
+
+.PHONY: functional-tests-run
+functional-tests-run: collectstatic  ## Run the functional tests
+	DJANGO_SETTINGS_MODULE=cms.settings.functional_test poetry run ./manage.py migrate --noinput
+	poetry run behave functional_tests
+
+.PHONY: functional-tests
+functional-tests: functional-tests-up functional-tests-run functional-tests-down  ## Run the functional tests with backing services (all in one)
+
+.PHONY: playwright-install
+playwright-install:  ## Install Playwright dependencies
+	poetry run playwright install --with-deps
```

Comment:
Suggestion: Add this after the `make install-dev` call so all dependencies are installed in 1 place?

Reply:
I avoided this because some existing CI jobs call `make install-dev` and I don't want to burden them with downloading browsers they don't need.

Reply:
Fair point, I'm happy to leave as is.

Reply:
I'll leave it for now as the Makefile is starting to get a bit bloated and it's easy enough to change down the line if we decide we want to.